lab3react/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Shouldn't have separate directories for different labs.  The branches take care of that.